framework/src/modules/chain/components/storage/sql/migrations/updates/20190410112400_add_asset_field_mem_accounts.sql

```diff
@@ -19,7 +19,7 @@
 */
 
  -- Add asset column to trs table as jsonb
-ALTER TABLE "mem_accounts" ADD COLUMN IF NOT EXISTS "asset" jsonb;
+ALTER TABLE "mem_accounts" ADD COLUMN IF NOT EXISTS "asset" jsonb NOT NULL DEFAULT '{}'::jsonb;
```

Comment:
If this PR is merged into `2.1`, updating from `2.0` to `2.1` won't apply this change.